docs/Reference/API-Methods.md

```diff
@@ -2970,6 +2970,53 @@ If the boolean value is `true`, the proposal is to add a signer. If `false`, the
     The `DEBUG` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) 
     or [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options to enable the `DEBUG` API methods.
 
+### debug_accountRange
+
+Returns the accounts for a specified block.
```

Comment:
Is debug_accountRange ever likely to be called directly?  Or is it the same as debug_storageRangeAt which states the following: 
Remix uses debug_storageRangeAt to implement debugging. Use the Debugger tab in Remix rather than calling debug_storageRangeAt directly.

Reply:
Added "Retesteth uses debug_accountRange to implement debugging."